app/src/main/java/com/example/taskthree/viewmodel/CounterViewModel.kt

```diff
@@ -5,17 +5,15 @@ import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 
 class CounterViewModel : ViewModel() {
-    private var counter = MutableLiveData<Int>()
+    private val counter = MutableLiveData<Int>()
+    val counterOberser: LiveData<Int>
+        get() = counter
 
     init {
         counter.value = 0
     }
 
-    fun getCounter(): LiveData<Int> {
-        return counter
-    }
-
     fun incrementCounter() {
         counter.value = (counter.value ?: 0) + 1
```

Comment:
Kod okunabilirligini arttirmak icin plus fonksiyonu kullanilabilir;
```suggestion
        counter.value = counter.value?.plus(1)
```